### ISSUE TITLE
svg support for generated plantuml images

### ISSUE THREAD

**Issue by volc** [jvantuyl/sublime_diagram_plugin#47]
Hi Jayson,

what do you think about adding an additional falg in the preferences for the output format of the rendered image? :) something like:

```
{
    "viewer" : "Preview", // Preview, QuickLook, EyeOfGnome, WindowsDefaultViewer
    "format" : "svg", // png, svg, ascii
    "check_on_startup": true,
    "charset": null // Can be "UTF-8" - to support non-latin text in diagram
}
```

the plantuml library already has the appropriate methods for outputting svg images :)

**Comment by jvantuyl** [comment]
I would merge a patch that implemented this.

**Comment by evandrocoan** [comment]
Sublime Text does not render SVG, so using it would not help much: 

1. https://github.com/SublimeTextIssues/Core/issues/556 No preview for svg

**Comment by jvantuyl** [comment]
The diagram plugin *does* support other viewers that do.  ¯\_(ツ)_/¯

**Comment by jvantuyl** [comment]
This was implemented in #59.
